lib/ts/recipe/emailpassword/types.ts

```diff
@@ -84,6 +84,7 @@ export type RecipeInterface = {
         }[];
         options?: RecipeFunctionOptions;
         userContext: any;
+        passwordToken?: string;
     }) => Promise<
         | {
```

Comment:
This PR is mostly good. A few missing changes:
- Please update the function signature in the emailpassword/index.ts file
- Also the thirdpartyemailpassword/index.ts and thirdpartyemailpassword/types.ts file

Reply:
Hey sorry I missed it let me update it @rishabhpoddar